conf.py

```diff
@@ -35,6 +35,7 @@
     'myst_parser',
     'sphinx_rtd_theme',
     'sphinx.ext.todo',
+    'sphinxcontrib.spelling',
```

Comment:
curious why you added this...was there a problem with the spell checking config?

Reply:
sphinx was demanding it on my machine .. wouldn't build otherwise ... (maybe something else was missing .. I fiddled around a good bit till `make` went through ..)

Reply:
update: I removed it and it still seems to build --> leaving it like that (without the change to conf.py) then .